### ISSUE TITLE
Does not work on 3.37.90

### ISSUE THREAD

**Issue by bubi-luka** [micheleg/dash-to-dock#1299]
After upgrading to Fedora 33 (development) and GNOME 3.37.90 this extension stopped working. I hope you will be able to make it work on Gnome 3.38. Thank you!

**Comment by UA27** [comment]
Can you link this in anyway to #1268 ? You should probably try out aimproxy's fork and see if it fixes it.

Also, probably same as #1258. If you can post logs/more info.

**Comment by timocapa** [comment]
> You should probably try out aimproxy's fork and see if it fixes it.

The fork makes dash-to-dock load again but it'll still be in a VERY VERY broken state and essentially be unusable.

**Comment by bubi-luka** [comment]
I believe it is the same as #1258. What logs, additional info do you need?

**Comment by bubi-luka** [comment]
I have tried aimproxy's fork and it mostly work. So far (for my use) I only noticed that 9 dots - Show Applications button is not working.

**Comment by UA27** [comment]
@bubi-luka the fork's author also mentioned the Show Applications button doesn't work anymore, so your issue was pretty much the same one he fixed, no need for additional info. Also, he had a lead on what might cause the Show Application button to fail and was waiting on some feedback I think, if you want to give it a go.

**Comment by 3v1n0** [comment]
@aimproxy, that's already handled in #1309 

Since #1301 has more activity, closing this as duplicate of that.

/duplicate #1301

**Comment by 3v1n0** [comment]
Duplicate of #1301

**Comment by mikedep333** [comment]
I am assuming you installed the extension manually rather than from RPM.

However, FYI, I submitted PR #1309 as as an update to the RPM "gnome-shell-extension-dash-to-dock" in Fedora 33:
https://bodhi.fedoraproject.org/updates/FEDORA-2020-9760c37431

I will update the Fedora 33 package as the PR is updated.

**Comment by micheleg** [comment]
New version with 3.38 support tagged (71abe80) and uploaded to the extensions website (still in review). Sorry for the delay.

**Comment by bubi-luka** [comment]
Working like a charm! Thank you!
